README.md

```diff
@@ -32,415 +32,450 @@
-* CSS custom properties are required to have two dashes before the name (--*), and can have any value;
-* They can then be picked up using `var(--foo)`, or `var(--foo, <fallback-value>)`
+* CSS custom properties are required to have two dashes before the name (`--*`), and can have any value.
+* They can be picked up using `var(--foo)`, or `var(--foo, <fallback-value>)`.
 
 ```html
-	<style>
-		div {
-			/* fallback if custom properties are not supported (IE11) */
-			color: green
-			--header-color: green;
-		}
-	</style>
-	<div id="div-host"></div> <!-- green -->
-	<span id="span-host"></span> <!-- red -->
-
-	<!-- Visual representation of the Shadow DOM -->
-	<style>
-		h1 {
-			color: var(--header-color, red);
-		}
-	</style>
-	<h1>Hello World</h1>
+<style>
+	div {
+		/* fallback if custom properties are not supported (IE11) */
+		color: green
+		--header-color: green;
+	}
+</style>
+<div id="div-host"></div> <!-- green -->
+<span id="span-host"></span> <!-- red -->
+
+<!-- Visual representation of the Shadow DOM -->
+<style>
+	h1 {
+		color: var(--header-color, red);
+	}
+</style>
+<h1>Hello World</h1>
 ```
 
 ## <a name='HTMLImport'></a>HTML Import
-* HTML imports allow fragments of HTML to be moved into another document. It does not need to be a full HTML page (with a `<head>`, `<body>`, etc) but may contain anything and all that is allowed within HTML, like markup content, script, styling and of course HTML imports of their own.
-* The mimetype of the import will be `text/html`
-* You can define an HTML import declaring a `<link rel="import">`
+* HTML imports allow fragments of HTML to be inserted into another document.
+  It does not need to be a full HTML page (with a `<head>`, `<body>`, etc) but may contain anything that is allowed within HTML, like markup content, script, styling, and HTML imports of their own, of course.
+* The MIME type of the import is `text/html`.
+* You can define an HTML import by declaring a `<link rel="import">`
  
 ```html
-	<head>
-		<!-- external resources need to be CORS enabled -->
-		<link rel="import" href="/path/to/imports/stuff.html">
-	</head>
+<head>
+	<!-- external resources need to be CORS enabled -->
+	<link rel="import" href="/path/to/imports/stuff.html">
+</head>
 ```
 
-* Or you can create one with javascript.
+* Or you can create one with JavaScript.
 
 ```javascript
-	var link = document.createElement('link');
-	link.rel = 'import';
-	link.href = 'file.html';
-	document.head.appendChild(link);
+const link = document.createElement('link');
+link.rel = 'import';
+link.href = 'file.html';
+document.head.appendChild(link);
 ```
 
-* An HTML import will not just add its content to the document, it will make it available for use later.
+* An HTML import will not just add its content to the document, it will make it available for later use, as well.
```

Comment:
The contents are not added at all, they are _only_ available for later use. I suggest removing "just" and "as well", and adding a "but".